package.json

```diff
@@ -50,8 +54,10 @@
     "envify": "^3.4.0",
     "eslint": "^3.3.1",
     "eslint-config-mourner": "^2.0.0",
+    "eslint-plugin-flowtype": "^2.20.0",
     "eslint-plugin-html": "^1.5.1",
     "github-slugger": "^1.1.1",
+    "flow-bin": "^0.2.3",
```

Comment:
👋 I'm just wondering why this is such an old version of Flow? Right now they're up to 0.32 now. Each time we upgrade Flow in Studio it's a little bit of an ordeal since lots of new errors show up.